### DISCUSSION TITLE
https://1kevinson.com/react-component-essentials-beginner-guide/

### DISCUSSION THREAD

**Discussion by giscus** [1kevinson/BLOG-TUTORIALS#20] [Announcements]
# https://1kevinson.com/react-component-essentials-beginner-guide/

In this blog post you will understand how React components work and how to create a simple React component

https://1kevinson.com/react-component-essentials-beginner-guide/

**Comment by defofotg** [comment]
Not knowing anything at all about React, this introduction is very useful. Thank you very much. You never disappoint :)

**Reply by 1kevinson** [reply]
Thanks for your review @defofotg

**Comment by xotoverse** [comment]
This article illustrates perfectly the little pieces of technologies required when creating React components. Its a really good place to come back to when you think you might also be missing something ! :) good work Kevin!

**Reply by 1kevinson** [reply]
Thanks for your review mate !
